dcos_launch/platforms/aws.py

```diff
@@ -211,6 +211,7 @@ class CfStack:
     def __init__(self, stack_name, boto_wrapper):
         self.boto_wrapper = boto_wrapper
         self.stack = self.boto_wrapper.resource('cloudformation').Stack(stack_name)
+        self.name = stack_name
```

Comment:
I need that information contained in a variable called "name" because the code in common.py uses resource_group.name in a lot of places and that code must work for every platform. Resource groups and deployments already had a member called "name" so I just added one to CfStack. Otherwise I need to do a get_name() method in each platform